myapp/app/models/task.rb

```diff
@@ -2,4 +2,6 @@
 
 class Task < ApplicationRecord
   validates :name, presence: true, length: { maximum: 50 }
+
+  enum status: %i[未着手 着手中 完了]
```

Comment:
このPRではステータスの確認・更新するところは無いのですが、
enumの値を動的メソッドに利用できるので、英名にしておくと後々使いやすくなるのでおすすめです！

```suggestion
  enum status: %i[not_started in_progress done]
```

ステータス確認: `task.in_progress?`
ステータス更新: `task.done!`

とかが出来ます。

参考: [Railsのenumを使いこなす方法（翻訳）](https://techracho.bpsinc.jp/hachi8833/2022_02_18/115735#:~:text=%E3%81%8D%E6%8F%9B%E3%81%88%E3%81%BE%E3%81%99%E3%80%82-,post%E3%81%AE%E3%82%B9%E3%83%86%E3%83%BC%E3%82%BF%E3%82%B9%E3%82%92%E7%85%A7%E5%90%88%E3%81%99%E3%82%8B,-Rails%E3%81%AF%E3%80%81%E7%89%B9%E5%AE%9A)

Reply:
おーなるほど！ありがとうございます！修正します！

Reply:
https://github.com/Fablic/training/pull/1276/commits/cb7e620641f3d729b77428e9d74288630952091d で対応しました！

合わせて、i18nの対応もしました -> https://github.com/Fablic/training/pull/1276/commits/95536dd40e9429d7796b59c43ac42280cb5bbaef

ご確認よろしくお願いいたします

Reply:
確認いたしました！
問題ないと思います！